lib/resources/postgres_session.rb

```diff
@@ -26,16 +27,23 @@ class PostgresSession < Inspec.resource(1)
     name 'postgres_session'
     desc 'Use the postgres_session InSpec audit resource to test SQL commands run against a PostgreSQL database.'
     example "
-      sql = postgres_session('username', 'password')
+      sql = postgres_session('username', 'password', 'host')
+      query('sql_query', ['database_name'])` contains the query and (optional) database to execute
+
+      # default values:
+      # username: 'postgres'
+      # host: 'localhost'
+      # db: databse == db_user running the sql query
 
       describe sql.query('SELECT * FROM pg_shadow WHERE passwd IS NULL;') do
         its('output') { should eq('') }
       end
     "
 
-    def initialize(user, pass)
+    def initialize(user, pass, host)
```

Comment:
This isn't backward compatible with the existing resource. We should probably change this to:

```
def initialize(user, pass, host=nil)
```

Otherwise, anyone already using a `postgres_session` resource will get:

```
ArgumentError: wrong number of arguments (given 2, expected 3)
```